ansible/roles/pico-shell/tasks/dependencies.yml

```diff
@@ -11,36 +11,46 @@
   apt:
     state: present
     name: [
-      'software-properties-common',
-      'xinetd',
-      'dpkg',
       'dpkg-dev',
+      'dpkg',
       'fakeroot',
-      'python-virtualenv',
+      'gcc-multilib',
+      'libffi-dev',
+      'libssl-dev',
+      'nginx',
+      'php7.0-cli',       # php5 package deprecated
+      'php7.0-sqlite3',   # used to serve shellinabox and challenge binaries
       'python-pip',       # used for pam module dependencies
-      'python3',          # used by shell_manager
+      'python-virtualenv',
       'python3-pip',      # used for shell_manager dependencies
-      'python3.6',        # from PPA, to cover all likely interpreter directives
       'python3.6-dev',
       'python3.6-venv',
-      'libffi-dev',
-      'libssl-dev',
+      'python3.6',        # from PPA, to cover all likely interpreter directives
+      'python3',          # used by shell_manager
+      'shellinabox',
       'socat',
-      'php7.0-cli',       # php5 package deprecated
-      'php7.0-sqlite3',   # used to serve shellinabox and challenge binaries
-      'gcc-multilib',
-      'nginx',
-      'shellinabox']
+      'software-properties-common',
+      'xinetd',
+      ]
 
 # Needed by templated challenge types
 - name: Install uwsgi and plugins for shell server
   apt:
-    name: ['uwsgi', 'uwsgi-plugin-php', 'uwsgi-plugin-python3', 'uwsgi-plugin-python', 'python-flask']
+    name: [
+        'python-flask',
+        'uwsgi-plugin-php',
+        'uwsgi-plugin-python',
+        'uwsgi-plugin-python3',
+        'uwsgi',
+        ]
     state: present
 
 - name: Install pam dependencies
   apt:
-    name: ['libpam-python', 'python-setuptools']
+    name: [
+        'libpam-python',
+        'python-setuptools'
```

Comment:
Missing comma? (I haven't checked the other lists; it just so happens that this one caught my eyes.)

Reply:
I had overlooked this... fixed in 2f60238f0c740d76023db83943ee73cf763b4f8a. Checked the others and each dependency list now definitely has trailing commas.